### PR TITLE
Home

### PR DESCRIPTION
**Pull request by ashishnnnnn**
-> Added Post Section On Home Page.
-> Replaced CSS with Tailwind

### CODE REVIEW

**Review by sharathnair9999 — APPROVED** [review]
Liked the usage of tailwind classes
Small suggestion  you could show the recently added posts to the top of the page rather than bottom

File: src/ApiCalls/addNewPost/addNewPost.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  try {
+    const response = await axios({
+      method: "POST",
+      url: "/api/posts",
+      data: { postData: post },
+      headers: {
+        authorization: loginToken,
+      },
+    });
+    console.log(response.data.posts);
```
Comment (ArjunGTX) [inline src/ApiCalls/addNewPost/addNewPost.js]: console.log can be removed

File: src/Components/BottomBar/BottomBar.jsx [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+  {
+    name: "Profile",
+    icon_class: "far fa-user",
+  },
+];
+
+export const BottomBar = () => {
+  const location = useLocation().pathname;
+  return (
+    <div className="border-2 md:hidden flex bg-slate-900		 p-4 gap-3  w-10/11	 rounded-3xl h-24	fixed	bottom-1 right-3 left-3 justify-around	">
```
Comment (ArjunGTX) [inline src/Components/BottomBar/BottomBar.jsx]: code can be formatted

**Review by ArjunGTX — APPROVED** [review]

File: src/Components/BottomBar/BottomBar.jsx (hunk 2 above)
Comment (ashishnnnnn) [reply src/Components/BottomBar/BottomBar.jsx]: Done

File: src/ApiCalls/addNewPost/addNewPost.js (hunk 1 above)
Comment (ashishnnnnn) [reply src/ApiCalls/addNewPost/addNewPost.js]: Done

**Pull request merged**